my-todo/src/reducers/reducer.js

```diff
@@ -0,0 +1,40 @@
+export const todo = [
+    {
+        item: 'Learn about reducers',
+        selectedDate: '2017-05-24T10:30',
+        completed: false,
+        id: 3892987589
+    }
+];
+
+
+export const reducer = (state, action) => {
+    console.log("Reducer State", state, "Reducer Action", action);
+    switch (action.type){
+        case "ADD_TODO":
+            console.log("Reducer Case State", state);
+            return [
+                ...state,
+                {
+                    item: action.payload,
+                    selectedDate: action.payload,
```

Comment:
here you have 1 payload, and you're setting item and date to the same payload.  You need to send in an object with 2 properties and set them separately